src/main/java/com/ty/loanApp/dao/implementation/LoanEnquiryStepProcessDaoImplementation.java

```diff
@@ -17,4 +20,36 @@ public LoanEnquiryStepProcess saveEnquiryStepProcess(LoanEnquiryStepProcess loan
 		return loanEnquiryStepProcessRepository.save(loanEnquiryStepProcess);
 	}
 
+	@Override
+	public LoanEnquiryStepProcess updateEnquiryStepProcess(int processId, LoanEnquiryStepProcess loanEnquiryStepProcess) {
+		Optional<LoanEnquiryStepProcess> op = loanEnquiryStepProcessRepository.findById(processId);
+		if(op.isPresent())
+		{
+			LoanEnquiryStepProcess enquiryStepProcess = op.get();
+			loanEnquiryStepProcess.setProcessId(enquiryStepProcess.getProcessId());
```

Comment:
pass the processId directly because its validated

Reply:
I have Resolved the Isuue